weave/trace_server/validation_util.py

```diff
@@ -58,6 +58,6 @@ def require_internal_ref_uri(
 
 
 def require_max_str_len(s: str, length: int) -> str:
-    if len(s) >= length:
+    if len(s) > length:
```

Comment:
we were truncating to 128 right? If so, this off by one fix will make the sentry error way less common presumably?

Reply:
we could add a test for this, that is **doesn't** throw when at 128

Reply:
adding a test